go/storage/client/init.go

```diff
@@ -55,25 +54,28 @@ func New(
 	ctx context.Context,
 	namespace common.Namespace,
 	ident *identity.Identity,
-	schedulerBackend scheduler.Backend,
 	registryBackend registry.Backend,
 	runtime registry.RuntimeDescriptorProvider,
-	extraWatcherOpts ...committee.WatcherOption,
 ) (api.Backend, error) {
-	watcherOpts := append(extraWatcherOpts, committee.WithAutomaticEpochTransitions())
-	committeeWatcher, err := committee.NewWatcher(
+	nl, err := nodes.NewRuntimeNodeLookup(
```

Comment:
The method comment above `// New creates a new storage client that automatically follows a given runtime's storage committee.`
should change, since this now follows all registered storage nodes for the runtime with RoleStorageRPC enabled. While before it followed the active committee members.

Note that: `go/runtime/registry` also uses this method, did we double-check that anything using the runtime registry will/should work with this change? 
e.g. if there are no storage nodes with `RoleStorageRPC` enabled, the storage part of the `go/runtime/registry` won't work.

Reply:
> Note that: go/runtime/registry also uses this method, did we double-check that anything using the runtime registry will/should work with this change?

AFAIK only the runtime client uses this constructor as storage and compute workers have their own committee handling.